source/developer/localization.md

```diff
@@ -16,10 +16,11 @@ To translate a language:
 
 #### Translations Updates
 
-1. *New* and *updated* strings are imported to the [Mattermost Translation Server](http://translate.mattermost.com) on a daily basis at **12:00 GMT**
-2. A new **Pull Request** is being made with the latest translations that reaches at least **Beta Quality** to the [Mattermost Platform Repo](https://github.com/mattermost/platform) every day at **22:00 GMT** without the need to do it on your own.
+1. **New** and **updated** strings are imported to the [Mattermost Translation Server](http://translate.mattermost.com) each day at **00:00 GMT**
```

Comment:
Not sure what the principle is on bolding here, seems a bit inconsistent. If you want to bold the dates/times for emphasis I'd propose:

1. Being consistent in the bullet point below as well.
2. Maybe also bold the day? 0/5 (ie `**each day at 00:00 GMT**`, `**each Monday at 22:00 GMT**`)